web_development_101/project_html_css.md

```diff
@@ -1204,6 +1204,7 @@ There are a couple of ways to go about doing this, but the simplest is this:
 * [Jason's solution (Easy)](https://github.com/jn4248/google-homepage) | [View in browser](https://jn4248.github.io/google-homepage/)
 * [Jason's solution (Difficult)](https://github.com/jn4248/google-results-page) | [View in browser](https://jn4248.github.io/google-results-page/)
 ### Additional Resources
+* [Datagoil's Solution (Easy & Difficult)] | [View in Browser (Easy)](https://datagoil.github.io/google-homepage/) | [View in Browser (Difficult)](https://datagoil.github.io/google-homepage-difficult/)
```

Comment:
Looks like you've added your solution under the additional resources heading. Can you put it above this please? 😄

Reply:
Of course. My bad. I created a new pull request. Was that the proper way to do it or should I have made the adjustment here in the review/comment area somehow?